dist/reporttable.js

```diff
@@ -62,7 +63,7 @@
             frozenIndex == -1 ? 
                 that.frozenIndices.push(index) : that.frozenIndices.splice(frozenIndex, 1);
 
-            $(".reporttable_header, .koddiTableGraph, .reporttable_footer").find("tr").each(function () {
+            that.frozenContainer.children().find("tr").each(function () {
```

Comment:
Same as previous comment, `children()` could be `find('table')` in order to be more descriptive.